src/pyff/builtins.py

```diff
@@ -960,6 +961,67 @@ def _discojson(req: Plumbing.Request, *opts):
     return json.dumps(res)
 
 
+@pipe(name='tinfojson')
```

Comment:
Well the discojson contains metadata for a list of IdPs, and this contains trust info for a list of SPs... The usage then in the MDQ server is different for each list - so to put both in the same structure I think we would want a pair of separated lists

Reply:
yeah that makes sense. Maybe we should make this really clear by having a discojson for IdPs and one for SPs because we really do need to have all the SPs available to the discovery UX anyway (displayname etc)